### ISSUE TITLE
how ploidyfix.vcf created?

### ISSUE THREAD

**Issue by junchend** [chapmanb/bcbio-nextgen#1248]
Dear chapman,

Thanks for the great job for bcbio, just a small problem when I check the command log file:

cat /NAS101/work/gatk-haplotype/P59-effects-ploidyfix.vcf  | /usr/local/bin/pbgzip -n 15  -c > /NAS101/work/gatk-haplotype/tx/tmp6y8uFn/P59-effects-ploidyfix.vcf.gz

how P59-effects-ploidyfix.vcf created? I can't find the command, before it is P59-effects.vcf.gz

Another problem is when I using metadata option, haplotypecaller  will include all the samples together to call the variants, time is consuming, so what's your opinion for calling variants using sample by sample or combining them in a list or using joint calling option?

Thanks again for your kindness.

Juncheng

**Comment by chapmanb** [comment]
Juncheng;
Sorry about the confusion with ploidyfix. That's run as python code so won't be present in the log file and the implementation is here:

https://github.com/chapmanb/bcbio-nextgen/blob/master/bcbio/variation/ploidy.py#L55

I pushed fixes that will only apply it if necessary -- female only batches where we remove Y chromosome calls. Previously it was being applied more generally but not changing things since the sex chromosome and mitochondrial calling has been moved to be part of ploidy, Hope this will help make things less confusing going forward.

This post describes the tradeoffs between joint, pooled and single sample calling in terms of quality:

http://bcb.io/2014/10/07/joint-calling/

There is some benefit to pooling calls together, especially in lower coverage regions and related populations where you expect them to share mutations. Using joint calling with GATK HaplotypeCaller will provide you this benefit while avoiding the extra overhead of calling samples together and might be a nice compromise. Hope this helps -- happy to follow up if you have more questions.
